tests/eas/acceptance: set sched_initial_test_util

bjackman opened the pull request:
This tunable is required to use the "sched" cpufreq governor (schedfreq)

Note that this isn't required right now - the acceptance tests use the performance governor - but we should expect people to want to change that.

derkling commented on tests/eas/acceptance.py:
```diff
@@ -54,15 +54,20 @@ def setUpClass(cls, *args, **kwargs):
-            except TargetError:
-                # That flag doesn't exist on mainline-integration kernels, so
-                # don't worry if the file isn't present.
+                cls.target.write_value(path, value)
```
Isn't just enough to use
```python
cls.target.write_value(path, value, verify=False)
```

Both target.write_value and target.executes provides verification flags which should silently ignore errors in case the specified path does not exist or is not writable (e.g. because of permissions).

derkling commented on tests/eas/acceptance.py:
```diff
@@ -54,15 +54,20 @@ def setUpClass(cls, *args, **kwargs):
                 pass
 
+        if SET_INITIAL_TASK_UTIL:
+            write_maybe("/proc/sys/kernel/sched_initial_task_util", 1024)
```
Given the previous comment we can probably avoid to add this write_maybe function...

bjackman replied:
Agreed

derkling commented on tests/eas/acceptance.py:
```diff
@@ -54,15 +54,20 @@ def setUpClass(cls, *args, **kwargs):
+        if SET_INITIAL_TASK_UTIL:
+            write_maybe("/proc/sys/kernel/sched_initial_task_util", 1024)
+
+        if SET_IS_BIG_LITTLE:
```
This option has not been added by this patch, maybe an error on chunks selection?

bjackman replied:
This `if` was already there, it just got reformatted :)

derkling commented:
Moreover: is this not something which is overlapping with the #209 PR by @credp... maybe we should merge this as well...

bjackman commented:
Pushed a new version addressing your review comments.

Re: The other PR - good point, I had forgotten about that! However I think this is still useful - I am now leaning towards making things explicit in Python (so you can see a call to `write_value`) rather than adding more stuff to our sparsely-documented JSON format. Do you and @credp agree with that suggestion?

derkling commented:
Ok, let's merge this and still pause the #209 since we are heading toward a more programmatic definition of tests. @credp, I'll keep #209 still open just to remember that we need these functions as well ;-)
